src/codewhisperer/service/inlineCompletionService.ts

```diff
@@ -168,6 +171,9 @@ class CodeWhispererInlineCompletionItemProvider implements vscode.InlineCompleti
             )
             this.nextMove = 0
             this._onDidShow.fire()
+            if (matchedCount >= 2 || RecommendationHandler.instance.hasNextToken()) {
```

Comment:
just for my understanding: what is this for?

Reply:
To highlight the Left right Arrow if there are >=2 suggestions